### PR TITLE
Bazaar plugin - Apply the title property to other components 

### PR DESCRIPTION
**Pull request by felipe-nardon**
## Hey, I just made a Pull Request!

Apply the title property to other components 

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [X] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/bazaar/api-report.md [hunk 1]
```diff
@@ -62,6 +64,7 @@ export const SortView: (props: SortViewProps) => React_2.JSX.Element;
 export type SortViewProps = {
   fullWidth?: boolean;
   fullHeight?: boolean;
+  codename: string;
```
Comment (vinzscam) [inline plugins/bazaar/api-report.md:67]:
```suggestion
  codename?: string;
```

if you make this optional then you could mark the changeset as `patch`

Comment (felipe-nardon) [reply plugins/bazaar/api-report.md]: Thanks for the sugestion

**Comment by vinzscam** [comment]
A better approach for delivering this feature would be to use the new internationalization features, which can potentially allow adopters to customize any text of the plugin. WDYT?
Here some docs: https://backstage.io/docs/plugins/internationalization/

**Comment by felipe-nardon** [comment]
> A better approach for delivering this feature would be to use the new internationalization features, which can potentially allow adopters to customize any text of the plugin. WDYT?
> Here some docs: https://backstage.io/docs/plugins/internationalization/

@vinzscam I18n implemented!

**Comment by felipe-nardon** [comment]
Hi @vinzscam!!

Any more questions?

**Pull request closed without merging**